readme.md

```
@@ -13,122 +13,383 @@
+    ```
+4.  **Levantar Base de Datos (Docker Compose):**
+    ```bash
+    Sin definir aun.
+    ```
+5.  **Ejecutar Migraciones y Seed en Prisma:**
+    ```bash
+    Sin definir aun.
+    ```
+6.  **Iniciar Servidores en modo Desarrollo:**
+    Desde la raíz del monorepo:
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)